### PR TITLE
Add round score visualization

### PR DESCRIPTION
**Pull request by ericgio**
Add a line graph showing players' round scores over time. This is an initial version; there are several things we could add, most notably some kind of interactive legend that allows users to modify which players show up.

image
image

### CODE REVIEW

File: src/components/Chart/RoundsChart.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+    return () => observer.unobserve(containerElem);
+  }, [containerRef]);
+
+  return size;
+};
+
+const translate = (x, y) => `translate(${x}, ${y})`;
+
+const COLORS = [
```
Comment (makinde) [inline src/components/Chart/RoundsChart.js:44]: If we have to add a player, do we have to add a color to this list? Not sure of how we can make that more obvious, if so. While it feels icky, players could have colors assigned in the PLAYERS constant. Or there could be code that automatically pics more colors.

File: src/components/Chart/RoundsChart.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  top: 16,
+};
+
+const parseTime = d3.timeParse('%Y-%m-%d');
+
+const RoundsChart = ({ data }) => {
+  const ref = useRef(null);
+  const { height, width } = useResizeObserver(ref);
+
+  const innerHeight = height - MARGIN.top - MARGIN.bottom;
```
Comment (makinde) [inline src/components/Chart/RoundsChart.js:67]: I can’t tell, but is this manual CSS handling? Would it make sense to have this live in a style sheet?

File: src/components/Chart/styles/RoundsChart.module.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+.axis {
+  path, line {
+    fill: none;
+    stroke: #dee2e6; // Match table border style
+    shape-rendering: crispEdges;
+  }
+
+  .domain {
```
Comment (makinde) [inline src/components/Chart/styles/RoundsChart.module.scss:12]: Is this class used? Can’t see it anywhere.

File: src/components/Chart/RoundsChart.js (hunk 1 above)
Comment (ericgio) [reply src/components/Chart/RoundsChart.js:44]: This is a static subset of colors from a d3 color scale. We can do it in a more dynamic way, this was more to get it working.

File: src/components/Chart/styles/RoundsChart.module.scss (hunk 3 above)
Comment (ericgio) [reply src/components/Chart/styles/RoundsChart.module.scss:12]: I'll remove.

File: src/components/Chart/RoundsChart.js (hunk 2 above)
Comment (ericgio) [reply src/components/Chart/RoundsChart.js:67]: It's not CSS; the height/width values are calculated based on the size of the container.

**Pull request merged**